src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.engine.backend
+
+import cromwell.binding.values.WdlValue
+import cromwell.binding.{Call, WdlStandardLibraryFunctions, WorkflowDescriptor}
+
+import scala.util.Try
+
+trait BackendCall {
```

Comment:
What I was suggesting in the local backend thing was something like this:

    trait BackendCall[A <: BackendCall[A]] {
      ...
      ...
      def execute(A): Try[...]
    }